packages/manager/modules/vps/src/dashboard/migrate/migrate.routing.js

```diff
@@ -32,5 +49,8 @@ export default /* @ngInject */ ($stateProvider) => {
         return promise;
       },
     },
+    atInternet: {
+      ignore: true,
```

Comment:
Maybe we could use the 
```suggestion
      rename: <trackingValue>,
```
instead

Reply:
I did try but i cannot inject the values which are `resolved` and i do not want to send the `planCodes` as `stateParams`. So i had to ignore here and add page tracking on `resolve/ngInit` instead.